src/components/Competition/ConfigManager/GeneralConfig/GeneralConfig.js

```diff
@@ -222,16 +223,30 @@ const GeneralConfig = ({ wcif, onWcifChange }) => {
               </Select>
             </FormControl>
           </Grid>
-          <FormControlLabel
-            control={
-              <Checkbox
-                name="localNamesFirst"
-                checked={localNamesFirst}
-                onChange={handleCheckboxChange}
-              />
-            }
-            label="Swap latin names with local ones"
-          />
+          <Grid item>
+            <FormControlLabel
+              control={
+                <Checkbox
+                  name="includeCoverSheets"
+                  checked={includeCoverSheets}
```

Comment:
Let's be more explicit just in case: `printScorecardsCoverSheets`.

Reply:
Good call!